apps/src/templates/sectionProgress/progressTables/sectionProgressTableStoryBuilder.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import ProgressTableDetailView from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableDetailView';
+import ProgressTableSummaryView from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableSummaryView';
+import {Provider} from 'react-redux';
+import {createStore, wrapTable} from '../sectionProgressTestHelpers';
+
+/**
+ * The variety of stories here can be useful during development, but add
+ * unnecessary work to our unit tests and have proven to be potentially flaky
+ * due to timeout while processing so much data. Set this value to `true` to
+ * enable all the stories.
+ */
+const INCLUDE_LARGE_STORIES = true;
```

Comment:
Should this be false?

Reply:
hah! yes. i was testing to make sure it worked before i pushed!